tests/by-util/test_dircolors.rs

```diff
@@ -159,6 +159,18 @@ fn test_quoting() {
         .no_stderr();
 }
 
+/*
+#[test]
+fn test_print_ls_colors() {
+    new_ucmd!()
+        .pipe_in("OWT 40;33\n")
+        .args(&["--print-ls-colors"])
+        .succeeds()
+        .stdout_is("\x1B[40;33mtw\t40;33\x1B[0m\n")
+        .no_stderr();
+}
+*/
```

Comment:
What's going on here? I'm not able to reproduce what this is supposed to do with GNU I think.

Reply:
This is tested in tests/misc/dircolors.pl

Reply:
reproduce this test?